services/get_account_details.proto

```diff
@@ -1,13 +1,26 @@
+/**
+ * # Get Account Details
+ * A standard query to inspect the full detail of an account.
+ *
+ * > REVIEW NOTE
+ * >> This query in NetworkAdmin service duplicates CryptoGetInfo in Token Service.<br/>
```

Comment:
@povolev15 You implemented this, do you have insight as to why we have these two duplicative queries?

Reply:
The biggest difference between CryptoGetInfo and NetworkGetAccountDetails is that NetworkGetAccountDetails is privileged, i.e. it must be signed by a superuser. However I'm not sure why privileged users can't use the non-privileged query too. Other differences:

CryptoGetAccountInfo response contains the following data which NetworkGetAccountDetails does not:
- long generateSendRecordThreshold
- long generateReceiveRecordThreshold
- List<LiveHash> liveHashes
- long ethereumNonce
- StakingInfo stakingInfo

Conversely, NetworkGetAccountDetails contains the following data which CryptoGetAccountInfo does not:
- List<GrantedCryptoAllowance> grantedCryptoAllowances
- List<GrantedNftAllowance> grantedNftAllowances
- List<GrantedTokenAllowance> grantedTokenAllowances

Reply:
The reason for adding `getAccountDetails` is to get allowances on an account. Since allowances can be almost 100 on each account, getAccountInfo will return too much data. So, we added a privileged query at that time. 

Now mirror node provides that information so users dont need to query from consensus node.

Reply:
Should/could we deprecate one or both of these?

Reply:
Probably that changes once Block Nodes and Block Streams are fully enabled, but for now we'll keep them both.